src/profile/Profile.tsx

```diff
@@ -20,15 +20,7 @@ export const Profile = () => {
   const { data } = useQuery(MeGQL);
 
   if (loading) return null;
-  if (!loading && !session)
-    return (
-      <>
-        <Text>log in to view your profile...</Text>
-        <Button mt="8" colorScheme="teal" onClick={() => signIn()}>
-          sign in
-        </Button>
-      </>
-    );
+  if (!session) return null; // shouldn't get here due to SSR
```

Comment:
Maybe we should pass the session in as props then if we're getting it in SSR instead of fetching it from the client side